src/app/pages/transactions/history/transactions-history-table-data-source.ts

```diff
@@ -371,12 +372,13 @@ export class TransactionsHistoryTableDataSource extends TableDataSource<Transact
     ];
     // Show Site Area Filter If Organization component is active
     if (this.componentService.isActive(TenantComponents.ORGANIZATION)) {
-      filters.push(new SiteTableFilter().getFilterDef());
-      filters.push(new SiteAreaTableFilter().getFilterDef());
+      const siteFilter = new SiteTableFilter(null, [issuerFilter]).getFilterDef();
+      filters.push(siteFilter);
+      filters.push(new SiteAreaTableFilter([issuerFilter, siteFilter]).getFilterDef());
     }
     if (this.authorizationService.isAdmin() || this.authorizationService.hasSitesAdminRights()) {
-      filters.push(new UserTableFilter(this.authorizationService.getSitesAdmin()).getFilterDef());
-      filters.push(new TagTableFilter().getFilterDef());
+      filters.push(new UserTableFilter(this.authorizationService.getSitesAdmin(), [issuerFilter]).getFilterDef());
```

Comment:
To remove this.authorizationService.getSitesAdmin().
Sites are now handled in the backend.
Test the Site Admin role btw to be sure.

Reply:
Tested: the authorisations is handled in the backend